src/components/Kbar/Actions.ts

```diff
@@ -0,0 +1,66 @@
+const actions = ({ setTheme }: { setTheme: (theme: string) => void }) => [
+  // CREATE CLIP REDIRECT
+  {
+    id: 'create',
+    name: 'Create clip',
+    shortcut: ['c'],
+    keywords: 'create clip',
+    section: 'Clip',
+    perform: () => window.location.replace('/'),
```

Comment:
I'd rather use the Next router for transitions without page reloads 👀 
https://nextjs.org/docs/api-reference/next/router#routerpush